client/components/edit/ObjectStatus.tsx

```diff
@@ -8,17 +9,20 @@ export interface ObjectStatusProps {
 }
 
 export const ObjectStatus = ({ pid }: ObjectStatusProps): React.ReactElement => {
+    const {
+        action: { toggleModal },
+    } = useGlobalContext();
     const {
         state: { objectDetailsStorage },
-        action: { setStateModalActivePid, toggleStateModal },
+        action: { setStateModalActivePid },
     } = useEditorContext();
     const loaded = Object.prototype.hasOwnProperty.call(objectDetailsStorage, pid);
     const details = loaded ? objectDetailsStorage[pid] : {};
 
     const stateTxt = details.state ?? "Unknown";
     const clickAction = () => {
         setStateModalActivePid(pid);
-        toggleStateModal();
+        toggleModal("state");
```

Comment:
Should this be openModal for consistency with other components, or is there a reason to use toggleModal?

Reply:
I addressed this in [f48b273](https://github.com/FalveyLibraryTechnology/VuDL/pull/256/commits/f48b273b4389299e6da8d7c39c3e2b29a1b77cec).